bin/node-gyp.js

```diff
@@ -131,9 +131,8 @@ function errorMessage () {
 
 function issueMessage () {
   errorMessage()
-  log.error('', ['This is a bug in `node-gyp`.',
-    'Try to update node-gyp and file an Issue if it does not help:',
-    '    <https://github.com/nodejs/node-gyp/issues>'
+  log.error('', ['Node-gyp failed to build your package.',
+    'Try to update node-gyp and if it does not help file an issue with the package author'
```

Comment:
"update node-gyp" is no trivial matter though, I think this was written even before it was bundled with npm. Maybe we should say "Try to update npm and/or node-gyp" or something along those lines? I don't think we can fit in explanations of how to make npm use an external node-gyp in this message and that's not necessarily good practice anyway.